packages/v4/src/pages/index.js

```diff
@@ -70,8 +72,10 @@ const IndexPage = ({ location }) => {
             {aboutPatternFly}
           </AutoLinkHeader>
           <Text component={TextVariants.p} className="ws-mdx-p">
-            We provide tools like design documentation, components, and code examples to make it possible for anyone to
-            design and build responsive, accessible user experiences.
+            We think quality product experiences should be easy to build and easy to scale. So we provide the tools to make that happen. 
```

Comment:
@abigaeljamie is the "So" supposed to start a different sentence? That caught me off guard in the implementation, but I understand if that new clause is supposed to separate ideas.

Reply:
It is supposed to be a new sentence, but it doesn't have to be! If it made you stumble, then we can merge it into the same sentence :)